packages/xo-server/src/api/xostor.mjs

```diff
@@ -116,18 +117,30 @@ export async function create({ description, disksByHosts, force, ignoreFileSyste
   const boundInstallDependencies = installDependencies.bind(this)
   await asyncEach(hosts, host => boundInstallDependencies({ host }), { stopOnError: false })
   const boundFormatDisks = formatDisks.bind(this)
-  await asyncEach(
-    hosts,
-    host => boundFormatDisks({ disks: disksByHosts[host.id], host, force, ignoreFileSystems, provisioning }),
-    {
-      stopOnError: false,
-    },
-  )
+  try {
+    await asyncEach(
+      hosts,
+      host => boundFormatDisks({ disks: disksByHost[host.id], host, force, ignoreFileSystems, provisioning }),
+      {
+        stopOnError: false,
+      }
+    )
+  } catch (error) {
+    log.error(error.errors[0])
+    const isForceRequired = error.errors.every(error => error.code === 'LVM_ERROR(5)')
```

Comment:
What if it's true for only some of them? Do we know what it would mean? Shouldn't we throw the `operationFailed` error in that case too?

Reply:
If all errors are not "LVM_ERROR(5)", this means that there were errors while formatting the disks that we cannot control today. In my opinion it is better to throw the error than to throw a failed operation. For me, Operation Failed should be started when it is known "why the operation failed". In this case the operation could have failed with completely different errors from each other and so it would seem strange to me to pass a global error to different errors.

In the future, as errors are discovered and understood, the code will likely change.

Reply:
That makes sense :)